### ISSUE TITLE
Add pa11y accessibility tests to vets.gov site

### ISSUE THREAD

**Issue by dannychapman** [department-of-veterans-affairs/vets-website#790]

**Comment by joduinn** [comment]
Found in triage. 

@dannychapman what is the status of this? Is this something we still want/need to do ?

**Comment by dannychapman** [comment]
@joduinn Thanks for following up on this. I think automated testing for accessibility should be something we implement. Whether it's pa11y or another tool is something we should access.

**Comment by MarinaMartin** [comment]
Please check with @orischwartz and @shanear as I think they have an automated tool called Sniffybara in place...

**Comment by plusjeff** [comment]
Using Attool instead
